src/hyperion/parameters/components.py

```diff
@@ -189,7 +235,7 @@ class OptionalGonioAngleStarts(BaseModel):
 
 
 class TemporaryIspybExtras(BaseModel):
-    # for while we still need ISpyB params - to be removed in #1277 and/or #43
+    # for while we still need ISpyB params_to be removed in #1277 and/or #43
```

Comment:
Nit: Why did this get changed?

Reply:
overeager find and replace :man_facepalming: